### DISCUSSION TITLE
Masa CMS 7.3

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#36] [Announcements]
Over the last few months we’ve been working hard on our first release and we’re proud to announce it’s ready for you to enjoy!

Masa 7.3 is the our release that features new functionality. We’ve added a complete replacement for CKFinder (that we had made optional in 7.2), improved user experience when editing content in the administrator, introduce the Masa Scope and prepared for a next major release by deprecating and notifying about deprecated functionality being used.

# New features

- New interface for editing content and components in the [Masa CMS Administrator](https://docs.masacms.com/content/adding-content/via-administrator/)
- New [File Browser](https://docs.masacms.com/content/managing-content/filebrowser/)
- File Browser enabled by default
- Introducing a [deprecation warnings module](https://docs.masacms.com/deprecations/deprecation_warnings/)
- Introducing the [Masa Scope](https://docs.masacms.com/front-end-layout-manager/m-tag/masa-scope/)
- ORM Assembler is now in BETA
- Changed the default theme to [MasaBootstrap5](https://github.com/MasaCMS/MasaBootstrap5)

# Bug fixes

- Old, unused code from Advertisement Manager removed from codebase
- Fixed a tenacious JavaScript console error
- ORM Scaffolder option list and option values list were incorrectly displayed

# Deprecations

- Deprecated Mura Scope and tag
- Deprecated Razuna integration
- Deprecated Simple Forms
- [Documented deprecations](https://docs.masacms.com/deprecations/list-of_deprecations/)
- [Documented deprecations inherited from Mura 7.1](https://docs.masacms.com/deprecations/mura-deprecations/)

**Comment by Tigerlady** [comment]
Thank you for adding a file manager.

**Comment by tbugler70** [comment]
Juat found MASA and I'm quite impressed - however with the deprecation of Simple Forms, what is the work around for creating ones own custom forms - I've always used simple forms in MURA as it gave me more control and flexibility with my forms.
